### PR TITLE
Update README details

### PR DESCRIPTION
**Pull request by sradco**
Update README details

### CODE REVIEW

**Comment by sradco** [comment]
@vladikr  had some more issues to fix. Please review.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ The query will be bound by the Virtual Machine lifecycle timeline.
 All imported log files will be streamed into an integrated instance of elasticsearch.
 
 The service operates on compressed collected must-gather files, but can be extended to work also with cluster dump or connect to a live cluster.
+
+Logsviewer also run insights rules to detect known issues and reports them if identified.
```
Comment (vladikr) [inline README.md:16]: s/run/runs

**Comment by vladikr** [comment]
Nice! Thanks

File: README.md (hunk 1 above)
Comment (vladikr) [inline README.md:16]:
```suggestion
Logsviewer also runs insights rules to detect known issues and reports them if identified.
```

**Review by vladikr — APPROVED** [review]

**Pull request merged**